deploy/contents/install/app/install-utils.sh

```diff
@@ -1493,6 +1511,12 @@ function is_service_requested {
     return $?
 }
 
+function is_deployment_type_requested {
```

Comment:
It seems that the logic of this function doesn't perform what is required, it seems that in current configuration it will return 0 only when you pass to it `classic`